### ISSUE TITLE
Questions about the cameras in the gazebo simulator

### ISSUE THREAD

**Issue by LucasLExcalibur** [BartlomiejKulecki/bebop2_multi_uav#1]
I noticed that in your drone files, while setting the machine_params, the "with_front_cam" was set to "false", which made no data in the ros topic "/bebop2/image_raw". But in your images, there seemed to have the camera data. If there is a way to start the camera to get the vision data in the ros package? And whether we could use multiple cameras of the drones in the gazebo simulation at the same time or not?

**Comment by BartlomiejKulecki** [comment]
Hello,
I used the cameras only when it was needed (by setting "with_front_cam" to "true"). For testing the flight, position control, etc. I turned it off because it speeded up the simulation. When I was using all cameras enabled the real-time factor in the Gazebo was low.

**Comment by LucasLExcalibur** [comment]
Hello,
Greatly thanks for your answering. When I tried to set more than one cameras in the Gazebo,  the loading process of the drones was too much slow to susccessfuly start the simulaiton. I could only one cameras in the way you mentioned. And I have another question is that if I want to use two or three cameras in the simulator, a computer with higher standard configuration whether would help make it go well?

**Comment by BartlomiejKulecki** [comment]
You can try to set low_gpu to true, but I don't know if it's enough to run more than one camera. A faster PC of course may help but I'm not sure it will solve the problem. You can search (or ask for) a solution at the Parrot developers forum.

**Comment by LucasLExcalibur** [comment]
Hello,
Thank you, your advice help me a lot! I tried the low_gpu setting you mentioned, that made me successfully insert two drones with cameras turning on in the simulator, although the image quality was not very good. I believe if I tried that in a PC with a higher configuration gpu, everything would go well! By the way, I also tried the TAP virtual network interface which is easliy available, that might provide some convenience in the multiple drones simulation. 
My problem has been well solvevd, I have been feeling great joy to communiacte with you, thanks again!
